mitiq/observable/observable.py

```diff
@@ -37,8 +39,8 @@ def __init__(self, *paulis: PauliString) -> None:
             paulis: PauliStrings used to define the observable.
 
         """
-        # TODO: Add option to Combine duplicates. E.g. [Z(0, Z(0)] -> [2*Z(0)].
         self._paulis = list(paulis)
+        self._combine_duplicates()
```

Comment:
I wasn't sure if its best to let this method modify `_paulis` in place or if I should move the method under some `utils` file and let it return a new list of paulis.

Modifying in place can be dangerous if gets used somewhere else within the class.

Reply:
I agree! 
It is safer if `_combine_duplicates()` is just some function from an input list of paulis to an output list of paulis.

Reply:
sounds good, there's no use of having this as a member method anymore, so I pulled it out of the class.